browser/resources/settings/brave_settings_overrides.js

```diff
@@ -117,17 +117,6 @@ BravePatching.RegisterPolymerComponentBehaviors({
 
 // Templates
 BravePatching.RegisterPolymerTemplateModifications({
-  'settings-ui': (templateContent) => {
```

Comment:
cc @petemill

Reply:
This part I did review with @petemill 😄 (needed his help actually, to make sure the mods I did were legit). My commit got smooshed into the one by @mkarolin